Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/test_data/get_workflows.json

```diff
@@ -0,0 +1,805 @@
+            "createdBy": {
+                "id": 1,
+                "domainId": 1,
+                "username": "firemon",
+                "firstName": "FireMon",
+                "lastName": "FireMon",
+                "displayName": "FireMon FireMon",
+                "email": "pushkar.ganpule@firemon.com",
+                "passwordExpired": false,
+                "locked": false,
+                "expired": false,
+                "enabled": true,
+                "lastLogin": "2021-08-18T08:38:54+0000",
+                "previousLogin": "2021-08-18T08:36:42+0000"
+            },
+            "lastModifiedDate": "2021-05-26T11:39:32+0000",
+            "lastModifiedBy": {
+                "id": 1,
+                "domainId": 1,
+                "username": "firemon",
+                "firstName": "FireMon",
+                "lastName": "FireMon",
+                "displayName": "FireMon FireMon",
+                "email": "pushkar.ganpule@firemon.com",
+                "passwordExpired": false,
+                "locked": false,
+                "expired": false,
+                "enabled": true,
+                "lastLogin": "2021-08-18T08:38:54+0000",
+                "previousLogin": "2021-08-18T08:36:42+0000"
+            },
+            "id": 17,
+            "workflow": {
+                "createdDate": "2021-04-26T07:10:46+0000",
+                "createdBy": {
+                    "id": 1,
+                    "domainId": 1,
+                    "username": "firemon",
+                    "firstName": "FireMon",
+                    "lastName": "FireMon",
+                    "displayName": "FireMon FireMon",
+                    "email": "pushkar.ganpule@firemon.com",
+                    "passwordExpired": false,
+                    "locked": false,
+                    "expired": false,
+                    "enabled": true,
+                    "lastLogin": "2021-08-18T08:38:54+0000",
+                    "previousLogin": "2021-08-18T08:36:42+0000"
+                },
+                "lastModifiedDate": "2021-05-26T11:39:32+0000",
+                "lastModifiedBy": {
+                    "id": 1,
+                    "domainId": 1,
+                    "username": "firemon",
+                    "firstName": "FireMon",
+                    "lastName": "FireMon",
+                    "displayName": "FireMon FireMon",
+                    "email": "pushkar.ganpule@firemon.com",
+                    "passwordExpired": false,
+                    "locked": false,
+                    "expired": false,
+                    "enabled": true,
+                    "lastLogin": "2021-08-18T08:38:54+0000",
+                    "previousLogin": "2021-08-18T08:36:42+0000"
+                },
+                "id": 1,
+                "domainId": 1,
+                "domainName": "Enterprise",
+                "name": "SN Workflow 11.1.8",
+                "pluginGroupId": "com.fm.wf.pp",

```

Comment:
Please go over all test data to verify there is no secrets or data you dont want him to be public.

Reply:
resolving as outdated